### ISSUE TITLE
How to setup Postgres database on local machine?

### ISSUE THREAD

**Issue by dahal** [envless/envless#157]
Please help us write tutorials and blog posts (it will be featured on Envless blog):

* [How to setup Postgres on Mac](https://github.com/envless/envless/blob/main/setup/postgres-on-mac.md)
* [How to setup Postgres on Linux](https://github.com/envless/envless/blob/main/setup/postgres-on-linux.md)
* [How to setup Postgres on Windows / WSL](https://github.com/envless/envless/blob/main/setup/postgres-on-windows.md)

**Comment by Aju100** [comment]
Assign on me for Mac one cc @dahal

**Comment by dahal** [comment]
@Aju100 please go ahead.

**Comment by dahal** [comment]
Chances are we don't need this anymore with docker, so I am closing this for now.

**Comment by Aju100** [comment]
Sure @dahal

**Comment by armichaud** [comment]
Hi @dahal , I found envless looking for `contributions-welcome` projects. I saw that the setup file for PostgreSQL on MacOS was inviting documentation and raised [this PR](https://github.com/envless/envless/pull/573) before I discovered this issue.

I'm inferring from the thread here that the preferred dev setup is containerized, but I was able to get the apps running more easily just running the server locally, so I think this documentation can't hurt.
